docs/source/fitting/fitfunctions/Meier.rst

```diff
@@ -11,7 +11,7 @@ Description
 
 Time dependence of the polarization function for a static muon interacting with nuclear spin [1].
 
-.. math:: A(t)=\frac13(2P_x+P_z)
+.. math:: A(t)=\frac{A_0*gau*Lor}{3}(2P_x+P_z)
```

Comment:
need to define what gau and lor are in the doc

Reply:
Ok, I think I can put the definitions for them as mathematical expressions

Reply:
Okay